libs/src/utils/network.js

```diff
@@ -27,7 +29,21 @@ async function timeRequests (requests) {
     timeRequest(request)
   ))
 
-  return timings.sort((a, b) => a.millis - b.millis)
+  timings.sort((a, b) => {
```

Comment:
We don't call this method anywhere else right? Just checking

Reply:
we don't call this anywhere else, but I think it would be a better idea to make this its separate method in case we want to have a separate method to time requests else where